pkg/apis/componentconfig/validation/validation_pluginargs.go

```diff
@@ -48,3 +48,12 @@ func validateCommonArgs(namespaces *api.Namespaces, labelSelector *metav1.LabelS
 
 	return nil
 }
+
+func ValidateRemoveDuplicatesArgs(args *componentconfig.RemoveDuplicatesArgs) error {
+	// At most one of include/exclude can be set
+	if args.Namespaces != nil && len(args.Namespaces.Include) > 0 && len(args.Namespaces.Exclude) > 0 {
```

Comment:
@a7i would you be fine keeping the validation as it is and refactoring the code after `ValidateNamespacesArgs` is introduced?

Reply:
i can bring this at toomanyrestart plugins pr at tonight. :)

Reply:
Yes, happy to.

Reply:
https://github.com/kubernetes-sigs/descheduler/pull/899

Reply:
so i need to wait #899 merge and rebase?

Reply:
Up to you.  I can rebase my PR if you end up merging first

Reply:
Ok,this pr still lack of lgtm label, so if your pr merge, i will rebase